sampler/xray/lib/opentelemetry/sampler/xray/version.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Copyright OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Sampler
+    module XRay
+      VERSION = '0.0.1'
```

Comment:
Our release tooling works best when a new gem is merged with version '0.0.0'

```suggestion
      VERSION = '0.0.0'
```